spec/requests/sessions/sign_out_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'DELETE api/v1/users/sign_out', type: :request do
+  let(:user) { create(:user) }
+  let(:req_headers) { auth_headers }
```

Comment:
You are not using this, as you are redefining it in both context, right?

Reply:
Oh you're right. Thanks!